target/bin/sedfile

```diff
@@ -11,19 +11,18 @@
 
 set -ueo pipefail
 
-HASHTOOL="sha1sum"
+HASHTOOL='sha1sum'
 SKIP_ERROR=0
 
 if [[ $# -lt 3 ]]
 then
-  echo "Error:  At least, three parameters must be given."
-  echo "Syntax: sedfile -i <replace/delete operation> <file>"
-  echo
+  echo 'At least three parameters must be given'
+  echo -e "Usage: ${0} -i <replace/delete operation> <file>\n"
   exit 1
 fi >&2
 
-[[ -f /CONTAINER_START ]] && SKIP_ERROR=1 # Hide error, if container was restarted.
-if [[ "${1}" == "--strict" ]]         # Show error every time.
+[[ -f /CONTAINER_START ]] && SKIP_ERROR=1 # hide error if container was restarted
+if [[ ${1} == '--strict' ]]                  # show error every time
```

Comment:
This should align with the comment above

```suggestion
if [[ ${1} == '--strict' ]]               # show error every time
```

Reply:
👍🏼

Reply:
Done in 4560650019f5e77321a6a46f825965bee51bd462